spec/app/ics-101-interchain-swap/README.md

```diff
@@ -16,20 +16,22 @@ This standard document specifies the packet data structure, state machine handli
 
 ### Motivation
 
-ICS-101 Interchain Swaps enables chains their own token pricing mechanism and exchange protocol via IBC transactions.  Each chain can thus play a role in a fully decentralised exchange network.
+ICS-101 Interchain Swaps enables chains their own token pricing mechanism and exchange protocol via IBC transactions. Each chain can thus play a role in a fully decentralised exchange network.
 
 Users might also prefer single asset pools over dual assets pools as it removes the risk of impermanent loss.
 
 ### Definitions
 
-`Interchain swap`: a IBC token swap protocol, built on top of an automated marketing making system, which leverages liquidity pools and incentives.  Each chain that integrates this app becomes part of a decentralized exchange network.
+`Interchain swap`: a IBC token swap protocol, built on top of an automated marketing making system, which leverages liquidity pools and incentives. Each chain that integrates this app becomes part of a decentralized exchange network.
 
-`Automated market makers(AMM)`: are decentralized exchanges that pool liquidity and allow tokens to be traded in a permissionless and automatic way.  Usually uses an invariant for token swapping calculation.  In this interchain standard, the Balancer algorithm is implemented.
+`Automated market makers(AMM)`: are decentralized exchanges that pool liquidity and allow tokens to be traded in a permissionless and automatic way. Usually uses an invariant for token swapping calculation. In this interchain standard, the Balancer algorithm is implemented.
 
 `Weighted pools`: liquidity pools characterized by the percentage weight of each token denomination maintained within.
 
 `Single-sided liquidity pools`: a liquidity pool that does not require users to deposit both token denominations -- one is enough.
```

Comment:
add this 
`But a single deposit will affect the price of the liquidity pool, increasing the opportunity for arbitrageurs`